space-framework/go.mod

```diff
@@ -11,54 +12,44 @@ require (
 	k8s.io/api v0.24.3
 	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v0.24.4
-	k8s.io/code-generator v0.24.3
```

Comment:
This one makes me nervous.  @ezrasilvera did you forget to do some hack to make this stick around (_does_ it need to stick around)?

Reply:
No I didn't. This is strange as tools.go needs it, so not sure why go mod tidy doesn't pick it up

Reply:
As this is in the space-framework, I suggest we merge the PR and if needed fix it in space specific PR